src/main/java/core/basesyntax/service/ReportContent.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.TypeOfFruit;
+import java.util.Map;
+
+public interface ReportContent {

```

Comment:
interface provides logic contract for class that doing something. Name should represent purpose of class/interface, not result
```suggestion
public interface ReportGenerator {
```